requirements/deploy.txt

```diff
@@ -120,9 +120,9 @@ django==4.2.14
     #   django-timezone-field
     #   djangorestframework
     #   jsonfield
-django-allauth[saml,socialaccount]==0.63.6
+django-allauth[saml,socialaccount]==64.0.0
```

Comment:
We should check if this breaks something -- cc @stsewd

Reply:
Nothing breaking for us https://docs.allauth.org/en/latest/release-notes/recent.html.